packages/cfd/src/Components/cfd-dxtrade-compare-content.tsx

```diff
@@ -0,0 +1,476 @@
+                                    }
+                                </Text>
+                            </div>
+                        </div>
+                    );
+                })}
+        </div>
+    );
+};
+
+const CfdDxtradeCompareContent = ({
+    landing_companies,
+    is_logged_in,
+    platform,
+    show_eu_related,
+    residence,
+    is_uk,
+}: TModalContentProps) => {
+    const [cols, setCols] = React.useState<Array<Record<string, string | React.ReactNode | undefined>>>([]);
+    const [template_columns, updateColumnsStyle] = React.useState('1fr 1.5fr');
+
+    React.useEffect(() => {
+        setCols(compareAccountsData({ landing_companies, is_logged_in, platform, show_eu_related, residence, is_uk }));
+        updateColumnsStyle('1fr 1.5fr');
+    }, [
+        landing_companies?.mt_financial_company,
+        landing_companies?.mt_gaming_company,
+        is_logged_in,
+        is_uk,
+        landing_companies,
+        platform,
+        show_eu_related,
+        residence,
+    ]);
+
+    const show_risk_message = !show_eu_related;
+
+    return (
+        <Div100vhContainer height_offset='40px' is_bypassed={isDesktop()}>
+            <ThemedScrollbars
+                className='cfd-compare-accounts'
+                style={{
+                    '--cfd-compare-accounts-template-columns': template_columns,
+                }}
+            >
+                <div className='cfd-compare-accounts__table-wrapper'>
+                    <Table className='cfd-compare-accounts__table'>
+                        <Table.Header>
+                            <Table.Row className='cfd-compare-accounts__table-row'>
+                                <Table.Head fixed />
+                                <Table.Head>{localize('Deriv X')}</Table.Head>
+                            </Table.Row>
+                        </Table.Header>
+                        <Table.Body>
+                            {cols.map((row, i) => (
+                                <Table.Row key={i} className='cfd-compare-accounts__table-row'>
+                                    {Object.keys(row).map((col, j) => (
+                                        <Table.Cell key={j} fixed={j === 0}>
+                                            {row[col]}
+                                        </Table.Cell>
+                                    ))}
+                                </Table.Row>
+                            ))}
+                        </Table.Body>
+                    </Table>
+                </div>
+                <CFDCompareAccountHint
+                    platform={platform}
+                    show_risk_message={show_risk_message}
+                    landing_companies={landing_companies}
+                    is_logged_in={is_logged_in}
+                    is_uk={is_uk}
+                />
+            </ThemedScrollbars>
+        </Div100vhContainer>
+    );
+};
+
+export default CfdDxtradeCompareContent;
```

Comment:
@hamid-deriv This file is [too big](https://refactoring.guru/smells/large-class) in my opinion, Can we break it into smaller components?